cmake/find/Findodb.cmake

```diff
@@ -0,0 +1,108 @@
+# ----------------------------------------------------------------------
```

Comment:
I saved your comments in my notes and will close this pull request to open a new one without the examples, as discussed previously, then I will look into the RPATH issue.